Mathlib/Algebra/Category/ModuleCat/Monoidal/Basic.lean

```diff
@@ -149,9 +149,6 @@ theorem triangle (M N : ModuleCat.{u} R) :
   ext : 1
   apply TensorProduct.ext_threefold
   intro x y z
-  -- Porting note (https://github.com/leanprover-community/mathlib4/pull/10934): used to be dsimp [tensorHom, associator]
-  change x ⊗ₜ[R] ((leftUnitor N).hom) (y ⊗ₜ[R] z) = ((rightUnitor M).hom) (x ⊗ₜ[R] y) ⊗ₜ[R] z
-  erw [TensorProduct.lid_tmul, TensorProduct.rid_tmul]
   exact (TensorProduct.smul_tmul _ _ _).symm
```

Comment:
This can be golfed to:
```lean
  intro x y
  exact TensorProduct.tmul_smul _ _
```

(And actually the `TensorProduct` isn't needed, but that should probably be done in another cleanup PR for this file.)

Thanks!
bors d+